keyless/pepper/common/src/vuf/bls12381_g1_bls.rs

```diff
@@ -14,11 +18,46 @@ use ark_std::{
     rand::{CryptoRng, RngCore},
     UniformRand,
 };
+use blstrs::{self, Compress};
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+use serde_big_array::BigArray;
 use std::ops::Mul;
 
 pub struct Bls12381G1Bls {}
 
-pub static DST: &[u8] = b"APTOS_PEPPER_SERVICE_BLS12381_VUF_DST";
+pub static DST: &[u8] = b"APTOS_PEPPER_BLS12381_VUF_DST";
+
+pub static PINKAS_DST: &[u8] = b"APTOS_PINKAS_PEPPER_DST";
+
+pub static PINKAS_SECRET_KEY_BASE_SEED: &[u8] = b"APTOS_PINKAS_PEPPER_SECRET_KEY_BASE_SEED";
+
+pub static PINKAS_SECRET_KEY_BASE_G2: Lazy<blstrs::G2Projective> =
+    Lazy::new(|| blstrs::G2Projective::hash_to_curve(PINKAS_SECRET_KEY_BASE_SEED, PINKAS_DST, b""));
+
+#[derive(Serialize, Deserialize, CryptoHasher, BCSCryptoHash)]
+pub struct PinkasPepper {
```

Comment:
Her we are under `vuf/bls12381_g1_bls` so "pepper" shouldn't appear here?
(while `vuf` is under `keyless/pepper`, it's designed to be generic and may be moved to aptos-crypto later...)

If you mean `PinkasVUF`, it's better to also put it in `vuf/bls12381_g1_pinkas` or something.

Reply:
Discussed offline, can refactor this later when we put in aptos-crypto